adam/curriculum/phase1_curriculum.py

```diff
@@ -143,7 +148,8 @@
 
 # Show each object once by itself
 def _make_each_object_by_itself_curriculum(
-    perception_generator: HighLevelSemanticsSituationToDevelopmentalPrimitivePerceptionGenerator = GAILA_PHASE_1_PERCEPTION_GENERATOR
+    perception_generator: HighLevelSemanticsSituationToDevelopmentalPrimitivePerceptionGenerator = GAILA_PHASE_1_PERCEPTION_GENERATOR,
+    language_generator=GAILA_PHASE_1_LANGUAGE_GENERATOR,
```

Comment:
Everywhere you're passing the language generator around is missing type annotations -- I'll mark every file needing this with 📓

Reply:
@LichMaster98 I omitted type annotations because there's two types, SimpleRuleBasedChineseLanguageGenerator and SimpleRuleBasedEnglishLanguageGenerator -- is there a way to type annotate despite this?